charts/tidb-backup/templates/scripts/_start_restore.sh.tpl

```diff
@@ -18,9 +18,23 @@ downloader \
   --bucket={{ .Values.ceph.bucket }} \
   --endpoint={{ .Values.ceph.endpoint }} \
   --srcDir=${BACKUP_NAME} \
-  --destDir=${dirname}
+  --destDir=/data
 {{- end }}
 
+set +e
```

Comment:
As @gregwebs said [here](https://github.com/pingcap/tidb-operator/pull/585/files#r294382884), the command in `until`, `if`, `when` will not exit when having errors. Will this work here too?

Reply:
It works in the diff. We should have `set -e` in all bash scripts, so this should be moved to the top.

Reply:
Yes. I've remove the `set +e` statement. I wonder where the problem occurred in the previous test environment.

Reply:
oh, sorry, didn't realize it was `+`. Yes, thanks for removing.